lib/kitchen/driver/ec2.rb

```diff
@@ -57,20 +66,20 @@ class Ec2 < Kitchen::Driver::Base # rubocop:disable Metrics/ClassLength
       default_config :price,              nil
       default_config :retryable_tries,    60
       default_config :retryable_sleep,    5
-      default_config :aws_access_key_id,  nil
+      default_config :aws_access_key_id,  ENV["AWS_SSH_KEY_ID"]
```

Comment:
I think I'm misunderstanding this change (and the one below in 72). The SSH key ID is not the same as the access_key. Can you help me understand this one?